plugin/signature_help.py

```diff
@@ -167,3 +188,59 @@ def _build_popup_content(self) -> str:
         return "\n".join(formatted)
+
+    def _build_popup_content_style_vscode(self) -> str:
+        # Fetch all the relevant data.
+        signature = {}  # type: Dict[str, Any]
+        signature_label = ""
+        signature_documentation = ""
+        parameter = {}  # type: Dict[str, Any]
+        parameter_label = ""
+        parameter_documentation = ""
+        if self._active_signature in range(0, len(self._signatures)):
+            signature = self._signatures[self._active_signature]
+            signature_label = html.escape(signature["label"], quote=False)
+            signature_documentation = signature.get("documentation", "")  # Optional.
+            parameters = signature.get("parameters", None)
+            if parameters and self._active_parameter in range(0, len(parameters)):
+                parameter = parameters[self._active_parameter]
+                parameter_label = html.escape(parameter["label"], quote=False)
+                parameter_documentation = parameter.get("documentation", "")  # Optional.
+
+        formatted = []
+
+        if len(self._signatures) > 1:
+            formatted.append(self._build_informational_header())
+
+        # Write the active signature and give special treatment to the active parameter (if found).
+        # Note that this <div> class and the extra <pre> are copied from mdpopups' HTML output. When mdpopups changes
+        # its output style, we must update this literal string accordingly.
+        formatted.append('<div class="highlight"><pre>')
+        if parameter_label:
+            signature_label = self._replace_active_parameter(signature_label, parameter_label)
+        formatted.append(signature_label)
+        formatted.append("</pre></div>")
+
+        if parameter_documentation:
+            formatted.append(parameter_documentation)
+
+        if signature_documentation:
+            formatted.append(signature_documentation)
+
+        return "\n".join(formatted)
+
+    def _replace_active_parameter(self, signature: str, parameter: str) -> str:
+        if parameter[0].isalnum() and parameter[-1].isalnum():
+            pattern = r'\b{}\b'.format(parameter)
+        else:
+            # If the left or right boundary of the parameter string is not an alphanumeric character, the \b check will
+            # never match. In this case, it's probably safe to assume the parameter string itself will be a good pattern
+            # to search for.
+            pattern = parameter
+        replacement = '<span style="font-weight: bold; text-decoration: underline">{}</span>'.format(parameter)
+        # FIXME: This is somewhat language-specific to look for an opening parenthesis. Most languages use parentheses
+        # for their parameter lists though.
+        index = signature.find('(')
+        # Note that this works even when we don't find an opening parenthesis, because .find returns -1 in that case.
+        string = signature[index + 1:]
```

Comment:
Please rename `string` and preferably `index` to explain what you are doing here.